api/v2/memo_relation_service.go

```diff
@@ -44,23 +46,26 @@ func (s *APIV2Service) SetMemoRelations(ctx context.Context, request *apiv2pb.Se
 
 func (s *APIV2Service) ListMemoRelations(ctx context.Context, request *apiv2pb.ListMemoRelationsRequest) (*apiv2pb.ListMemoRelationsResponse, error) {
 	relationList := []*apiv2pb.MemoRelation{}
-	tempList, err := s.Store.ListMemoRelations(ctx, &store.FindMemoRelation{
-		MemoID: &request.Id,
-	})
+	tempList, err := s.Store.V2.MemoRelation.
```

Comment:
Should we wrap a method in the `store` package?

Reply:
that's also my first try wrapping the db inside store.

but there are a few concerns, and we can discuss:
1. using the origin store interface has no way to distinguish v1 or v2, we can only replace the origin v1 directly in each function step by step.
2. it seems not worth creating a new store interface only for ent operations
3. the ent client seems to be easy enough to use